lib/src/rewrite.rs

```diff
@@ -766,8 +766,11 @@ pub struct DuplicateCommitsStats {
 /// The roots of `target_commits` are duplicated on top of the new
 /// `parent_commit_ids`, whilst other commits in `target_commits` are duplicated
 /// on top of the newly duplicated commits in the target set. If
-/// `children_commit_ids` is not empty, the `children_commit_ids` will be
-/// rebased onto the heads of the duplicated target commits.
+/// `target_descriptions` is not empty, it will be consulted to retrieve the new
+/// descriptions of the target commits, falling back to the original if the map
+/// does not contain an entry for a given commit. If `children_commit_ids` is
```

Comment:
nit: I would split this to new paragraph. Here we're discussing about the graph structure.